tests/unit/fem/test_pa_coeff.cpp

```diff
@@ -144,4 +156,207 @@ TEST_CASE("pa_coeff")
    }
 }
 
+TEST_CASE("Hcurl pa_coeff")
+{
+   for (dimension = 2; dimension < 4; ++dimension)
+   {
+      for (int coeffType = 0; coeffType < 2; ++coeffType)
+      {
+         for (int integrator = 0; integrator < 2; ++integrator)
+         {
+            const int ne = 2;
+            std::cout << "Testing " << dimension << "D ND partial assembly with "
+                      << "coeffType " << coeffType << " and "
+                      << "integrator " << integrator << std::endl;
+            for (int order = 1; order < 5; ++order)
+            {
+               Mesh* mesh;
```

Comment:
Not that it would make a big difference but you could move the mesh creation to the outermost loop.